basic_network_test.go

```diff
@@ -48,15 +48,10 @@ func setupNodes(t *testing.T, p1, p2 int) (*BasicVideoNetwork, *BasicVideoNetwor
 
 func connectHosts(h1, h2 host.Host) {
 	h1.Peerstore().AddAddrs(h2.ID(), h2.Addrs(), peerstore.PermanentAddrTTL)
-	h2.Peerstore().AddAddrs(h1.ID(), h1.Addrs(), peerstore.PermanentAddrTTL)
 	err := h1.Connect(context.Background(), peerstore.PeerInfo{ID: h2.ID()})
 	if err != nil {
 		glog.Errorf("Cannot connect h1 with h2: %v", err)
 	}
-	err = h2.Connect(context.Background(), peerstore.PeerInfo{ID: h1.ID()})
```

Comment:
Is the removal here because as long as there is a connection h1 -> h2, there does not need to be a connection in the other direction as well (at least for these tests)?

Reply:
Yep.  I think the h2->h1 connection is redundant.